README.md

```diff
@@ -24,3 +24,16 @@ Depending on your environment, you may find that this assignment is unnecessary.
 If you've installed Antora globally using Yarn, you may need to add `"$(yarn global dir)/node_modules"` to the `NODE_PATH` environment variable instead.
 
 Please read the [documentation](https://github.com/Mogztter/antora-lunr#enable-the-search-component-in-the-ui) to enable the search component in your documentation UI.
+
+## Upgrading
+
+When installing this generator globally (i.e., `npm i -g`), it also installs a private copy of Antora (minus the CLI).
+Thus, to upgrade the copy of Antora this site generator uses, you need to reinstall this package.
+Upgrading Antora alone isn't sufficient.
+
+To upgrade this site generator and Antora, open a terminal and type the following commands:
+
+    $ npm uninstall -g antora-site-generator-lunr @antora/cli @antora/site-generator-default
+      npm i -g antora-site-generator-lunr @antora/cli @antora/site-generator-default
+
+These commands ensure you have the latest version of both the site genernator and Antora regardess of whether you're using this site generator.
```

Comment:
```suggestion
These commands ensure you have the latest version of both the site generator and Antora regardless of whether you're using this site generator.
```

Reply:
Is there a rule about "that" in the sentence "These commands ensure that..." or "These commands ensure ...". When should we use it or not use it?

Reply:
It's a style thing. If you prefer brevity, you can drop it. If you need to fill space, include it. Both ways are equally correct grammatically.